workflows/cloudify_system_workflows/deployment_environment.py

```diff
@@ -158,6 +168,11 @@ def create(ctx, labels=None, inputs=None, skip_plugins_validation=False,
     idd.create(ctx, client, deployment_plan)
 
 
+def _get_deployments(client: 'CloudifyClusterClient',
+                     deployment_id: str) -> 'ListResponse':
```

Comment:
and ListResponse is probably not great either. I think we can do something like Iterable[Deployment]